pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,822 @@
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a6"
+    >
+      If you're using USWDS Gulp
+    </button>
+  </h4>
+  <div id="m-a6" class="usa-accordion__content usa-prose">
+    <ol>
+      <li>Search for a line like <strong>const USWDS = "node_modules/uswds/dist"</strong> in your Gulp setup. This indicates that you're using the Gulp setup we distributed as USWDS Gulp.</li>
+      <li>
+        Update the USWDS <strong>const</strong> to the updated uswds package location
+        <p><strong>Old</strong></p>
+          <code>const USWDS = "node_modules/uswds/dist";</code>
+        <p><strong>New</strong></p>
+          <code>const USWDS = "node_modules/@uswds/uswds";</code>
+          <br>
+      </li>
+      <li> 
+        Search for <strong>includePaths</strong> in your project's Gulp files. The paths in this list are where the Sass compiler looks for your source files. In USWDS 3.0
+        <p><strong>Old</strong></p>
+        <pre><code>.pipe(
+  sass({
+    includePaths: [
+      PROJECT_SASS_SRC,
+      `${USWDS}/scss`,
+      `${USWDS}/scss/packages`,
+    ],
+  })
+)</code></pre>
+        <p><strong>New</strong></p>
+        <pre><code>.pipe(
+  sass({
+    includePaths: [
+      PROJECT_SASS_SRC,
+      `${USWDS},
+      `${USWDS}/packages`,
+    ],
+  })
+)</code></pre>
+      </li>
+      <li>Recompile your Sass as usual. When it compiles, it is now using USWDS 3.0!</li>
+    </ol>
+  </div>
+
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a7"
+    >
+      If you're using USWDS Compile
+    </button>
+  </h4>
+  <div id="m-a7" class="usa-accordion__content usa-prose"  markdown="1">
+```
+/* gulpfile.js */
+
+const uswds = require("@uswds/compile");
+
+/**
+ * USWDS version
+ */
+
+uswds.settings.version = 3;
+```
+  </div>
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a8"
+    >
+      If you're using a custom Gulp workflow
+    </button>
+  </h4>
+  <div id="m-a8" class="usa-accordion__content usa-prose">
+    TK   
+  </div>
+
+  <h4 class="usa-accordion__heading">
+    <button
+      class="usa-accordion__button"
+      aria-controls="m-a9"
+    >
+      If you're using Webpack
+    </button>
+  </h4>
+  <div id="m-a9" class="usa-accordion__content usa-prose">
+    TK   
+  </div>
+</div>
+
+### 5. Update to Sass module syntax
+
+{:.border-top-2px.border-base-lighter.padding-top-1}
+USWDS 3.0 supports the deprecated Sass import syntax. You don't need to change to the new syntax in your project Sass to use USWDS 3.0. But once you update, we recommend updating to the new syntax as well.  You’ll need to both switch @import to @use, and then create @use project-specific settings.  In USWDS 3.0 with Sass modules, instead of having multiple theme files that contain every setting available to the design system, you only keep track of the settings you changed in a single file. 
```

Comment:
```suggestion
USWDS 3.0 supports the deprecated Sass `@import` syntax. You don't need to change to the new syntax in your project Sass to use USWDS 3.0. But once you update, we recommend updating to the new syntax as well.  You’ll need to both switch `@import` to `@use`, and then create `@use` project-specific settings.  In USWDS 3.0 with Sass modules, instead of having multiple theme files that contain every setting available to the design system, you only keep track of the settings you changed in a single file. 
```

Reply:
I like this! Should we do this instead of **bold** in other sections? This is right below it in that ordered list

![image](https://user-images.githubusercontent.com/25211650/164545589-ac5dddc4-f658-489f-be2e-dee557c4394a.png)

Reply:
I think we should be consistent throughout for all code references. Looking at the migration 2 guide, it looks like we **bold** keywords and `backtick` code references.

Reply:
Resolved in https://github.com/uswds/uswds-site/pull/1529/commits/86c766e2b9bfa4ad1497469c23aed00451ee0350